src/main/java/it/gov/pagopa/taxonomy/model/TaxonomyObjectStandard.java

```diff
@@ -0,0 +1,39 @@
+package it.gov.pagopa.taxonomy.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
+public class TaxonomyObjectStandard {
```

Comment:
it should extend TaxonomyObject, ignoring `combinazioneTopicSubtopic` and `flagNuovaCombinazione`